lib/rom/csv/commands/create.rb

```diff
@@ -0,0 +1,31 @@
+require 'rom/commands'
+require 'rom/commands/create'
+
+module ROM
+  module CSV
+    module Commands
+      class Create < ROM::Commands::Create
+        def execute(tuples)
+          insert_tuples = Array([tuples]).flatten.map do |tuple|
```

Comment:
I believe `Array([tuples]).flatten` is not needed, `Array(tuples).flatten` does the same. Moreover this flatten can be removed as well, but not sure about that.